ehr/resources/web/ehr/data/ClinicalEncountersClientStore.js

```diff
@@ -97,15 +100,26 @@ Ext4.define('EHR.data.ClinicalEncountersClientStore', {
         }
     },
 
-    getProcedureRecord: function(procedureId){
-        var procedureStore = EHR.DataEntryUtils.getProceduresStore();
-        LDK.Assert.assertNotEmpty('Unable to find procedureStore from ClinicalEncountersClientStore', procedureStore);
+    getProcedureRecordFromStore(procedureStore, procedureId){
         var procRecIdx = procedureStore.findExact('rowid', procedureId);
         LDK.Assert.assertTrue('Unable to find procedure record in ClinicalEncountersClientStore for procedureId: [' + procedureId + ']', procRecIdx > -1);
 
         var procedureRec = procedureStore.getAt(procRecIdx);
         LDK.Assert.assertNotEmpty('Unable to find procedure record from ClinicalEncountersClientStore.  ProcedureId was: [' + procedureId + ']', procedureRec);
 
         return procedureRec;
+    },
+
+    getProcedureRecord: function(procedureId, modifiedFieldNames, cb){
```

Comment:
A little doc here on the expected arguments for the callback function would be nice, and a note that it may be invoked synchronously or async

Reply:
added a comment